app/main.py

```diff
@@ -1,3 +1,68 @@
+from typing import Any
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Any) -> "Distance":
+        if isinstance(other, Distance):
+            return Distance(km=self.km + other.km)
+        elif isinstance(other, (int, float)):
+            return Distance(km=self.km + other)
+
+    def __iadd__(self, other: Any) -> "Distance":
+        if isinstance(other, Distance):
+            self.km += other.km
+        elif isinstance(other, (int, float)):
+            self.km += other
+        return self
+
+    def __mul__(self, other: Any) -> "Distance":
+        if isinstance(other, (int, float)):
+            return Distance(km=self.km * other)
+        elif isinstance(other, Distance):
+            raise TypeError("Cannot multiply two Distance instances")
+        raise TypeError("Division is only supported with int or float")
+
+    def __truediv__(self, other: Any) -> "Distance":
+        if other == 0:
+            raise ZeroDivisionError("Cannot divide by zero.")
+        elif isinstance(other, (int, float)):
+            return Distance(km=round(self.km / other, 2))
+
+    def __lt__(self, other: Any) -> bool:
```

Comment:
The return type annotation for the `__lt__` method should be `bool` instead of a string. Update it to `bool`.